Amazon_schema.txt

```diff
@@ -0,0 +1,60 @@
+create table users(userid int primary key identity(1,1),
+Name varchar(50),
+phone_number nvarchar(15),
+email varchar(254) unique,
+addressid int not null,
+foreign key(addressid) references addresses(addressid));
+
+
+create table addresses(addressid int primary key identity(1,1),
+StreetAddress VARCHAR(100),
+City VARCHAR(50),
+State VARCHAR(50),
+PostalCode VARCHAR(10),
+Country VARCHAR(50));
+
+
+create table products(productid int primary key identity(1,1),
+productname varchar(50) not null,
+categoryid int not null,
+price decimal(10,2) not null,
+foreign key(categoryid) references categories(categoryid));
+
+
+create table categories(categoryid int primary key identity(1,1),
+categoryname varchar(50) not null);
+
+
+create table orders(orderid int primary key identity(1,1),
+userid int not null,
+productid int not null,
```

Comment:
how to add multiple products

Reply:
create table OrderedProducts(OrderedProductsId int primary key identity(1,1),
ProductId int not null,
Quantity int not null,
Amount decimal(10,2) not null,
foreign key(ProductId) references Products(ProductId)
);


create table Orders(OrderID int primary key identity(1,1),
UserId int not null,
OrderedProductsId int not null,
TotalAmount decimal(10,2) not null,
OrderStatus VARCHAR(20) NOT NULL, 
OrderDate DATETIME NOT NULL DEFAULT GETDATE(),
foreign key(UserId) references Users(UserId));
foreign key(OrderedProductsId) references OrderedProducts(OrderedProductsId);

modified orders table and created another table named OrderedProducts